### DISCUSSION TITLE
Slowroll distribution

### DISCUSSION THREAD

**Discussion by armync** [agama-project/agama#1653] [Ideas]
Do you have any plans to add the **Slowroll** distribution in the selection menu?

I saw that other distros already started to be implemented (e.g. #1371 #1423). Nevertheless, Slowroll is an experimental product as far as I gather. Yet, it has some beta released entries as per https://download.opensuse.org/slowroll/iso/

Thanks!

**Comment by imobachgs** [comment]
No specific plans yet but we could coordinate with Slowroll release managers in case they are interested. I guess we can do it once Leap 16.0 is properly supported.

**Comment by shundhammer** [comment]
IMHO it would be just a clone of the setup for TW with adjusted repos. Plus the Slowroll logo, of course. ;-)

**Reply by imobachgs** [reply]
Please, see #1674 😃

**Comment by imobachgs** [comment]
Support for Slowroll has been added (see #1674 and #1684) thanks to @WesfunOfficial. Please, bear in mind that Slowroll is an experimental distribution only available in x86_64 architectures.

Thanks for opening the topic.
